caikit_nlp/resources/pretrained_model/base.py

```diff
@@ -209,16 +209,16 @@ def load(cls, model_path: str) -> Type["PretrainedModelBase"]:
         )
         return res
 
-    def save(self, model_path: str):
+    def save(
+        self, model_path: str, tokenizer_dirname: str = "", base_model_dirname: str = ""
```

Comment:
I think it shouldn't break anything, since this `save` function isn't getting used directly currently and since `load` function of this base class isn't using those dirnames directly, it won't break loading of these saved model as well. 

Idea behind this change was to let user of this module direct how the artifacts can be saved and changed the defaults, since the usual convention with the transformer models is to save both tokenizer and models in same directory. Having them saved as `""` would make the models we save with this function directly loadable via HF load function as well (since there won't be a separate path inside the folder).